src/index.js

```diff
@@ -159,15 +159,33 @@ function sanitizeHtml(html, options, _recursing) {
     }
   });
 
-  var depth = 0;
-  var stack = [];
-  var skipMap = {};
-  var transformMap = {};
-  var skipText = false;
-  var skipTextDepth = 0;
+  var depth;
+  var stack;
+  var skipMap;
+  var transformMap;
+  var skipText;
+  var skipTextDepth;
+
+  var initializeState = function() {
```

Comment:
Follow style of the module, use a named function and place it lower down, top-down programming.